pyinteraph/centrality_analysis.py

```diff
@@ -0,0 +1,604 @@
+                        default = b_default,
+                        help = b_helpstr)
+
+    d_choices = edge
+    d_default = "edge"
+    d_helpstr = f"Sort edge centralities. Use the name of the" \
+                f"desired measure. The name must match one of the names " \
+                f"used in option -c. (default: {d_default})."
+    parser.add_argument("-d", "--sort-edge",
+                        dest = "sort_edge",
+                        choices = d_choices,
+                        default = d_default,
+                        help = d_helpstr)
+
+    w_default = False
+    w_helpstr = f"Use edge weights to calculate centrality measures. " \
+                f"(default: {w_default})."
+    parser.add_argument("-w", "--use_weights",
+                        dest = "weight",
+                        action = "store_true",
+                        default = w_default,
+                        help = w_helpstr)
+
+    n_default = True
+    n_helpstr = f"Normalize centrality measures. " \
+                f"(default: {n_default})."
+    parser.add_argument("-n", "--normalize",
+                        dest = "norm",
+                        action = "store_true",
+                        default = n_default,
+                        help = n_helpstr)
+
+    e_default = False
+    e_helpstr = f"Use endpoints when calculating centrality measures. " \
+                f"(default: {e_default})."
+    parser.add_argument("-e", "--use_endpoints",
+                        dest = "endpoint",
+                        action = "store_true",
+                        default = e_default,
+                        help = e_helpstr)
+
+    k_default = 3
+    k_helpstr = f"The minimum cutoff for a node to be considered a hub. " \
+                f"(default: {c_default})."
+    parser.add_argument("-k", "--hub-cutoff",
+                        dest = "hub",
+                        default = k_default,
+                        type = int,
+                        help = k_helpstr)
+
+    g_helpstr = f"List of residues used for group centrality calculations. " \
+                f"e.g. A32,A35,A37:A40. Replace chain name with '_' if no " \
+                f"reference PDB file provided. e.g. _42,_57."
+    parser.add_argument("-g", "--group",
+                        dest = "group",
+                        default = None,
+                        type = str,
+                        help = g_helpstr)
+
+    o_default = "centrality"
+    o_helpstr = f"Output file name for centrality measures " \
+                f"(default: {o_default}.txt)."
+    parser.add_argument("-o", "--centrality-output",
+                        dest = "c_out",
+                        default = o_default,
+                        help = o_helpstr)
+
+    p_default = False
+    p_helpstr = f"For each centrality measure calculated, create a PDB file " \
+                f"where the bfactor column is replace by the centrality value." \
```

Comment:
Add a black space at the end of this string, otherwise, it will merge with the first word of the next line.